node/src/actors/chain_manager/transaction_factory.rs

```diff
@@ -213,7 +245,8 @@ mod tests {
         own_pkh: PublicKeyHash,
         all_utxos: &UnspentOutputsPool,
     ) -> Result<Transaction, NoMoney> {
-        let drt_tx = build_drt(dr_output, fee, own_utxos, own_pkh, all_utxos)?;
+        let timestamp = get_timestamp() as u64;
```

Comment:
We don't need the real timestamp in tests, just something `> 0`.
```suggestion
        let timestamp = 1;
```